transformer.go

```diff
@@ -0,0 +1,13 @@
+// Copyright (C) 2017 ScyllaDB
+// Use of this source code is governed by a ALv2-style
+// license that can be found in the LICENSE file.
+
+package gocqlx
+
+// Transformer transforms the value of the named parameter to another value.
+type Transformer func(name string, val interface{}) interface{}
+
+// DefaultBindTransformer just do nothing.
```

Comment:
It sounds a bit strange but I couldn't come up with anything better :)

Reply:
I set it as nil to save compatibility with older versions. Maybe someone uses insert/update statements with nil values to explicitly (or implicitly, who knows) delete columns